newTest.ts

```diff
@@ -0,0 +1,138 @@
+const users : [{"id": Number,"name": String}] = require("./data/users"); //Import users.json in data folder. 'require' will always load the JSON data as a JavaScript object
```

Comment:
If we are using this in fetching all the users, let's read this from the file in that API alone.